sql/database.sql

```diff
@@ -8761,9 +8761,9 @@ CREATE TABLE `x12_partners` (
   `x12_receiver_id` varchar(255) default NULL,
   `processing_format` enum('standard','medi-cal','cms','proxymed','oa_eligibility','availity_eligibility') default NULL,
   `x12_isa01` VARCHAR( 2 ) NOT NULL DEFAULT '00' COMMENT 'User logon Required Indicator',
-  `x12_isa02` VARCHAR( 10 ) NOT NULL DEFAULT '          ' COMMENT 'User Logon',
+  `x12_isa02` VARCHAR( 16 ) NOT NULL DEFAULT '          ' COMMENT 'User Logon',
```

Comment:
x12 may have a limit on these sizes. did you check both x12 spec and the x12 generator in x12 partners(may limit size to be sent there also)? 
Also you've reverted previous updates. Need to merge current database and upgrade scripts then add your changes.

Reply:
hi @juggernautsei , check out this [thread](https://community.open-emr.org/t/officeally-integration/13970/8?u=stephenwaite)

Reply:
@stephenwaite the person that I made the change for is using Availity. When she entered the username and password were truncated. I understand if the ISA has a fixed length in some manual somewhere but in practice, it seems the clearinghouse is not adhering to the 10 characters "standard". I will go ahead and drop this repo @sjpadgett

Reply:
@sjpadgett I will let you know if the other parts of the system reject the longer username and password. I will still drop this repo so it don't go stale.

Reply:
hi @juggernautsei , the problem is that the `ISA` record needs to be a fixed length so that some properties are set at certain fields further out in the record, like the delimiter.

Reply:
@stephenwaite I accept what you are saying. I don't know enough about this section. I was just trying to help. You guys got this.

Reply:
I'm curious about how these fields are getting treated by various clearinghouses like Availity, Change, and OA. Thanks @juggernautsei